examples/direct_fidelity_estimation.py

```diff
@@ -95,9 +95,74 @@ def compute_characteristic_function(circuit: cirq.Circuit,
     return sigma_i
 
 
+def _estimate_pauli_traces_clifford(n_qubits, clifford_state,
```

Comment:
type annotations?

Reply:
I know this is a private function, but this is an example so a little docstring would be helpful.

Reply:
Done and done.

I also took the liberty of doing the same for the other private function _estimate_pauli_traces_general().